ListLinked/LinkedList.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace LinkedList
+{
+    public  class LinkedList<T> : IEnumerable<T>
+    {
+        private int position = 0;
+        public Node<T> head;
+
+
+        public void Add(T value)
+        {
+            if (head == null)
+            {
+                head = new Node<T>(value);
+                return;
+            }
+            Node<T> current = head;
+            while (current.next != null)
+            {
+                current = current.next;
+            }
+            current.next = new Node<T>(value);
+
+        }
+
+        public void AddAt(int index, T value)
+        {
+            Node<T> current = head;
+            int head_position = 0;
```

Comment:
In c# we use camel case convention for local vars. So it should be headPosition.